servers/quarkus-server/src/main/resources/resource-config.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
We only add RocksDB to nessie-quarkus, but not nessie-lambda (local database doesn't make sense for a lambda)